src/lib/util/util.ts

```diff
@@ -454,7 +454,7 @@ export async function createMuteRole(message: Message) {
 }
 
 export function inlineCodeblock(input: string) {
-	return `\`${input.replace(/ /g, '\u00A0').replace(/`/g, '\\`')}\``;
+	return `\`${input.replace(/ /gs, '\u00A0').replace(/`/g, '\\`')}\``;
```

Comment:
Just curiosity, what's the effect of using the sticky flag here and how does that fix the warning?